sklearn/preprocessing/tests/test_data.py

```diff
@@ -642,12 +725,12 @@ def test_standard_scaler_trasform_with_partial_fit():
     for i, batch in enumerate(gen_batches(X.shape[0], 1)):
 
         X_sofar = X[:(i + 1), :]
+
         chunks_copy = X_sofar.copy()
         scaled_batch = StandardScaler().fit_transform(X_sofar)
 
         scaler_incr = scaler_incr.partial_fit(X[batch])
         scaled_incr = scaler_incr.transform(X_sofar)
-
```

Comment:
Please do not add or remove blank lines far away from your main code change to keep the diff of the PR focused on the main topic.